tests/kafkatest/services/kafka/kafka.py

```diff
@@ -642,7 +644,7 @@ def get_offset_shell(self, topic, partitions, max_wait_ms, offsets, time):
         if partitions:
             cmd += '  --partitions %s' % partitions
 
-        cmd += " 2>> /mnt/get_offset_shell.log | tee -a /mnt/get_offset_shell.log &"
+        cmd += " &>> %s/get_offset_shell.log &" % KafkaService.PERSISTENT_ROOT
```

Comment:
I looked at this again, and it appears that tee writes to stdout, in addition to appending to the file.  So I'm going to keep this as-is rather than changing it.